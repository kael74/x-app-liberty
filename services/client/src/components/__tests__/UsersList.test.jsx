import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import UsersList from '../UsersList';

const users = [
  {
    'active': true,
    'email': 'leandrosimeon@upeu.edu.pe',
    'id': 1,
    'username':'leandro'
  },
  {
    'active': true,
    'email': 'cetaceta@upeu.edu.pe',
    'id': 2,
    'username':'cetaceo'
  }
];

test('UsersList renders properly', () => {
  const wrapper = shallow(<UsersList users={users}/>);
  const element = wrapper.find('h4');
  expect(element.length).toBe(2);
  expect(element.get(0).props.children).toBe('leandro');
});

test('UsersList renders a snapshot properly', () => {
  const tree = renderer.create(<UsersList users={users}/>).toJSON;
  expect(tree).toMatchSnapshot();
});
