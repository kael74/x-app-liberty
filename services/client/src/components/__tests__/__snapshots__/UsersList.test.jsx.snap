// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders a snapshot properly 1`] = `[Function]`;
